src/components/TotalSupply.tsx

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "react-query"
+import { useNetwork } from "wagmi";
+import { MORPHS_NFT_CONTRACT_ADDRESSES } from "../constants/contracts";
+import { getGraphClient } from "../lib/graph";
+
+export default () => {
+  const [{ data: network }] = useNetwork();
+
+  const chainId = network.chain?.id;
+  const collection = MORPHS_NFT_CONTRACT_ADDRESSES[chainId ?? 1];
+  const { data, isLoading } = useQuery(['total supply', chainId], async () => {
+    const client = getGraphClient(chainId);
+    const resp = await client.collectionDetail({ address: collection });
+    return resp.data?.collection;
+  });
```

Comment:
example of use query + the graphql client

`client` is fully typed and surfaces all `.graphql` queries in the project